v0.3/tutorials/quickstart clock.eve

```diff
@@ -0,0 +1,145 @@
+---
+layout: default
+title: Quickstart + Counter
+---
+
+{% raw %}
+
+# Quickstart + Counter
+
+This guide expands on the quickstart by adding a live clock to the `Clock` page.
+
+## Quickstart code
+
+This is made to add onto the web app you made in the quickstart. You'll want to add the new code from here into the quickstart.eve document if you've already got the web app written there. Otherwise, you can quickly copy and paste the code in this section to recreate it. Whichever path you take, jump down to the `Building the clock` section to begin working on the counter!
+
+Header
```

Comment:
This one could use more instruction. It's not clear to me what the story is past this point.